docs/project/goals.md

```diff
@@ -244,13 +245,13 @@ features that encourage unchangeable constructs. For example, any feature with a
 contract that cannot be strengthened or weakened without breaking the expected
 usage patterns is inherently hostile to refactoring. Analogously, features or
 conventions that require simultaneously updating all users of an API when
-extending it are inherently hostile towards long-term maintenance of software.
+extending it are inherently hostile toward the long-term maintenance of software.
 
 **Support maintaining and evolving the language itself for decades.** We will
 not get the design of most language features correct on our first, second, or
 73rd try. As a consequence, there must be a built-in plan and ability to move
-Carbon forward at a reasonable pace and with a reasonable cost. Simultaneously,
-an evolving language must not leave software behind to languish, but bring
+Carbon forward at a reasonable pace and at a reasonable cost. Simultaneously,
+an evolving language must not leave software behind to languish but bring
```

Comment:
```suggestion
an evolving language must not leave software behind to languish, instead bringing
```

Comma is definitely correct here. Suggesting a slight phrasing change to emphasize contrast.